index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Shingirai Portfolio</title>
+</head>
+
+<body>
+    <header>
+        <a href="/" class="header-logo">
+            <h3>
+                Welcome
+            </h3>
+        </a>
+        <nav>
+            <img src="./assets/Union.svg" alt="menu link">
+        </nav>
+    </header>
+    <section class="intro">
+        <h1>
+            Hey there. I’m Shingirai
+        </h1>
+        <h1>
+            <span>I’m a software developer</span>
+        </h1>
+        <p>
+            I can help you build a product , feature or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded,
+            don’t hestiate to contact me.
+        </p>
+
+        <ul class="social-media">
+            <li>
+                <a href="">
+                    <img src="./assets/social-media/twitter.svg" alt="twitter icon">
+                </a>
+            </li>
+            <li>
+                <a href="">
+                    <img src="./assets/social-media/linkedin.svg" alt="linkedin icon">
+                </a>
+            </li>
+            <li>
+                <a href="">
+                    <img src="./assets/social-media/medium.svg" alt="medium icon">
+                </a>
+            </li>
+            <li>
+                <a href="">
+                    <img src="./assets/social-media/github.svg" alt="github icon">
+                </a>
+            </li>
+            <li>
+                <a href="">
+                    <img src="./assets/social-media/angellist.svg" alt="angellist icon">
+                </a>
+            </li>
+        </ul>
+
+        <button type="button">
+            <img src="./assets/Disabled.svg" alt="disabled button">
+        </button>
+    </section>  
```

Comment:
Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![aa](https://user-images.githubusercontent.com/69638013/158758997-2c4523e8-9dbc-4674-89b8-10eb5ad2535f.png) | ![bg](https://user-images.githubusercontent.com/69638013/158757622-3d19a6ae-8991-4d81-84ec-baf362e4f709.png)

 [ ] Kindly add the right background image to this section, use CSS and not img tags 👍🏿